### PR TITLE
fix(docs): token description updated

### LINKED ISSUE #62: Update ISLM token description

### PR DESCRIPTION
**Pull request by PastaHash**
Closes: #62 

## Description

- token description in basics section updated
- link to islamiccoin website added

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: docs/basics/tokens.md [hunk 1]
```diff
@@ -14,7 +14,9 @@ Haqq is a Cosmos SDK-based chain with full EVM support. Because of this architec
 
 Accounts can own SDK coins in their balance, which are used for operations with other Cosmos modules and transactions. Examples of these are using the coins for staking, IBC transfers, governance deposits and EVM  
 
-### ISLM
+### Islamic Coin (ISLM)
+
+[Islamic Coin](https://islamiccoin.net) (ISLM) is a native currency of Haqq
```
Comment (onsails) [inline docs/basics/tokens.md]: dot

File: docs/basics/tokens.md [hunk 2]
```diff
@@ -14,7 +14,9 @@ Haqq is a Cosmos SDK-based chain with full EVM support. Because of this architec
 
 Accounts can own SDK coins in their balance, which are used for operations with other Cosmos modules and transactions. Examples of these are using the coins for staking, IBC transfers, governance deposits and EVM
```
Comment (onsails) [inline docs/basics/tokens.md]: dot

**Review by onsails — CHANGES_REQUESTED** [review]

File: docs/basics/tokens.md (hunk 1 above)
Comment (PastaHash) [reply docs/basics/tokens.md]: fixed

File: docs/basics/tokens.md (hunk 2 above)
Comment (PastaHash) [reply docs/basics/tokens.md]: fixed

**Review by onsails — APPROVED** [review]

**Review by kioqq — APPROVED** [review]

**Pull request merged**